### ISSUE TITLE
仓库更新报告 (2025-03-25)

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#4]
# 仓库更新报告 (2025-03-26)

## vllm-project/vllm

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-03-25 22:50:09+00:00 | Varun Sundar Rabindranath | [Core] LoRA: V1 Scheduler optimization (#15422) |
| 2025-03-25 21:27:22+00:00 | Chenyaaang | [core] add bucket padding to tpu_model_runner (#14995) |
| 2025-03-25 21:22:26+00:00 | Lu Fang | [V1] Support long_prefill_token_threshold in v1 scheduler (#15419) |
| 2025-03-25 21:21:36+00:00 | Nick Hill | [V1][Minor] Use `SchedulerInterface` type for engine scheduler field (#15499) |
| 2025-03-25 20:43:54+00:00 | Nicolò Lucchesi | [TPU][V1] Fix Sampler recompilation (#15309) |
| 2025-03-25 20:31:08+00:00 | Maximilien de Bayser | Add workaround for shared field_names in pydantic model class (#13925) |
| 2025-03-25 19:00:32+00:00 | Joe Runde | [bugfix] add supports_v1 platform interface (#15417) |
| 2025-03-25 17:59:25+00:00 | Antonio Gómez | [Bugfix] Support triton==3.3.0+git95326d9f for RTX 5090 (Unsloth + vLLM compatibility) (#15471) |
| 2025-03-25 16:27:16+00:00 | yarongmu-google | [CI/Build] Add tests for the V1 tpu_model_runner. (#14843) |
| 2025-03-25 14:05:39+00:00 | youkaichao | [bugfix] fix inductor cache on max_position_embeddings (#15436) |
| 2025-03-25 13:50:49+00:00 | Szymon Ożóg | [Kernel] Fix conflicting macro names for gguf kernels (#15456) |
| 2025-03-25 11:01:58+00:00 | Cyrus Leung | [Doc] Update V1 user guide for multi-modality (#15460) |
| 2025-03-25 10:27:57+00:00 | Cyrus Leung | [Misc] Remove redundant `num_embeds` (#15443) |
| 2025-03-25 10:22:52+00:00 | Cyrus Leung | [Misc] Clean up MiniCPM-V/O code (#15337) |
| 2025-03-25 10:15:14+00:00 | Md. Shafi Hussain | Dockerfile.ppc64le changes to move to UBI (#15402) |
| 2025-03-25 09:34:59+00:00 | Thien Tran | [Kernel][CPU] CPU MLA (#14744) |
| 2025-03-25 08:43:00+00:00 | Siyuan Liu | [Hardware][TPU][Bugfix] Fix v1 mp profiler (#15409) |
| 2025-03-25 07:36:45+00:00 | Lu Fang | Fix CUDA kernel index data type in vllm/csrc/quantization/gptq_marlin/awq_marlin_repack.cu +10 (#15160) |
| 2025-03-25 04:04:41+00:00 | Woosuk Kwon | [V1][Spec Decode] Update target_logits in place for rejection sampling (#15427) |
| 2025-03-25 04:02:33+00:00 | Russell Bryant | [V1] guidance backend for structured output + `auto` fallback mode (#14779) |
| 2025-03-25 03:48:08+00:00 | Chauncey | [Bugfix] Fixed the issue of not being able to input video and image simultaneously (#15387) |
| 2025-03-25 03:43:51+00:00 | Tyler Michael Smith | Revert "Fix non-contiguous input passed to Marlin kernel (#15319)" (#15398) |
| 2025-03-25 03:43:48+00:00 | Jee Jee Li | [Misc] Remove LoRA log (#15388) |
| 2025-03-25 02:41:45+00:00 | Harry Mellor | Add pipeline parallel support to `TransformersModel` (#12832) |
| 2025-03-25 02:09:04+00:00 | Woosuk Kwon | [Minor][Spec Decode] Remove compiled_softmax (#15416) |
| 2025-03-25 00:16:46+00:00 | Woosuk Kwon | [V1][Spec Decode] Enable spec decode for top-p & top-k sampling (#15063) |

## sgl-project/sglang

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-03-25 23:53:12+00:00 | fzyzcjy | Add retry for flaky tests in CI (#4755) |
| 2025-03-25 09:01:21+00:00 | fzyzcjy | Fix warmup error when dp=1 (#4753) |
| 2025-03-25 08:17:03+00:00 | fzyzcjy | Fix test_expert_distribution failure (#4752) |
| 2025-03-25 07:44:01+00:00 | fzyzcjy | Revert "Add DeepEP tests into CI (#4737)" (#4751) |
| 2025-03-25 05:15:25+00:00 | DarkSharpness | [Feature] Support "strict" in function calling (#4310) |
| 2025-03-25 04:57:58+00:00 | HandH1998 | add dsv3 int8 test (#4705) |
| 2025-03-25 04:34:19+00:00 | yuhsaun-t | Add endpoints to dump selected expert ids (#4435) |
| 2025-03-25 04:08:05+00:00 | fzyzcjy | Speedup warmup when DP > 1 (#4695) |
| 2025-03-25 04:06:58+00:00 | lukec | fix FlashMLA cudagraph config (#4691) |
| 2025-03-25 04:05:45+00:00 | fzyzcjy | Super tiny fix typo (#4738) |
| 2025-03-25 03:56:31+00:00 | Chunan Zeng | [Benchmark] tilelang vs deepgemm vs w8a8_block_fp8_matmul (#4735) |
| 2025-03-25 03:53:23+00:00 | Yineng Zhang | support cu128 sgl-kernel (#4744) |
| 2025-03-25 03:08:40+00:00 | Mick | model: Minicpmo (#3023) |
| 2025-03-25 02:54:31+00:00 | fzyzcjy | Add DeepEP tests into CI (#4737) |
| 2025-03-25 02:50:23+00:00 | Trevor Morris | Support FP4 gemm (1/2) (#3899) |
| 2025-03-25 02:13:44+00:00 | Ximingwang-09 | [Model] Support Qwen2ForSequenceClassification (#4609) |
| 2025-03-25 01:07:08+00:00 | Stefan He | Fix circular imports in gptq.py and unblock test explorer (#4736) |
| 2025-03-25 00:31:19+00:00 | Ke Bao | Fix EAGLE3 for llama3.3 70b (#4716) |
